README.md

```diff
@@ -58,3 +41,35 @@ WinPcapUtils.capture_on("*Ethernet*", packet_callback)
 "\Device\NPF_{EAF47DBE-5B49-1337-1337-BD059E02666B} Microsoft 0 0"
 "\Device\NPF_{2997B9BB-AA53-1337-1337-B862F874271C} Microsoft 0 0"
 ```
+
+### Easy Packet live callback
+```python
+from winpcapy import WinPcapUtils
+
+# Example Callback function to parse IP packets
+def packet_callback(win_pcap, param, header, pkt_data):
+    # Assuming IP (for real parsing use modules like dpkt)
+    ip_frame = pkt_data[14:]
+    # Parse ips
+    src_ip = ".".join([str(ord(b)) for b in ip_frame[0xc:0x10]])
+    dst_ip = ".".join([str(ord(b)) for b in ip_frame[0x10:0x14]])
+    print("%s -> %s" % (src_ip, dst_ip))
+
+WinPcapUtils.capture_on("*Ethernet*", packet_callback)
+```
+
+```python
+from winpcapy import WinPcapUtils
+
+# Example Callback function to parse IP packets
+def packet_callback(win_pcap, param, header, pkt_data):
+    # Assuming IP (for real parsing use modules like dpkt)
+    ip_frame = pkt_data[14:]
+    # Parse ips
+    # Maybe you should use 'str(b)' instead of 'str(ord(b))'
```

Comment:
To get the IpV4 address you need to use ord.
Check the differences below between the two cases
```
>>> str("\xff")
'\xff'
>>> str(ord("\xff"))
'255'

```